src/pages/transaction-signing/components/transaction-errors.tsx

```diff
@@ -55,7 +55,7 @@ export const StxTransferInsufficientFundsErrorMessage = memo(props => {
               <Caption>
                 {balances?.stx?.balance
                   ? stacksValue({
-                      value: balances?.stx?.balance,
+                      value: balances?.stx?.balance.toString(),
```

Comment:
[with this change](https://github.com/blockstack/stacks-wallet-web/pull/1520/files#diff-3bdf507371227437baae41250f7cdcb2ce68fe4b4701d18904117edf7f86d13fR14), you can do this
```suggestion
                      value: balances?.stx?.balance,
```